.circleci/config.yml

```diff
@@ -1,56 +1,66 @@
-version: 2
+version: 2.1
 workflows:
-  version: 2
   test:
     jobs:
-      - test-7.0.1
-      - test-8.0.1
+      - test:
+          matrix:
+            parameters:
+              keycloak-version:
+                # 4.8.3 doesn't have a docker image
```

Comment:
There is docker image, only the version number is different, it is `4.8.3.Final`
in full `jboss/keycloak:4.8.3.Final`

https://hub.docker.com/layers/jboss/keycloak/4.8.3.Final/images/sha256-0ad4869e7fb5aa8c5d8e7ecc5866b1d3a8a4dc826843f40bc29bb8dae8e87efe?context=explore****